js/ui/grid_core/ui.grid_core.header_filter_core.js

```diff
@@ -197,12 +192,12 @@ export const HeaderFilterView = modules.View.inherit({
         this._popupContainer && this._popupContainer.$content().empty();
     },
 
-    _initializePopupContainer: function(options) {
+    _initializePopupContainer: function(options, headerFilterOptions) {
         const that = this;
         const $element = that.element();
-        const headerFilterOptions = that.option('headerFilter');
-        const width = options.headerFilter && options.headerFilter.width || headerFilterOptions && headerFilterOptions.width;
-        const height = options.headerFilter && options.headerFilter.height || headerFilterOptions && headerFilterOptions.height;
+        const width = headerFilterOptions.width;
+        const height = headerFilterOptions.height;
```

Comment:
Minor comment.

Let's use destructuring here:
```suggestion
        const { width, height } = headerFilterOptions;
```